src/main/java/gregtech/common/metatileentities/MetaTileEntities.java

```diff
@@ -512,6 +513,10 @@ public static void init() {
 
         CLEANROOM = registerMetaTileEntity(1035, new MetaTileEntityCleanroom(gregtechId("cleanroom")));
 
+        if(ConfigHolder.machines.largeminer.extremeLargeMiner) {
+            EXTREME_LARGE_MINER = registerMetaTileEntity(1036, new MetaTileEntityLargeMiner(gregtechId("large_miner.zpm"), GTValues.ZPM, ConfigHolder.machines.largeminer.largeMinerSpeed[3], ConfigHolder.machines.largeminer.largeMinerChunkRadii[3], ConfigHolder.machines.largeminer.largeMinerFortune[3], ConfigHolder.machines.largeminer.largeMinerMace[3], Materials.HSSE, ConfigHolder.machines.largeminer.largeMinerFluidConsume[3]));
```

Comment:
We don't want super-high-tier ore miner in this version of GregTech
You want to remove it or Tech will reject this PR...